CHANGELOG.rst

```diff
@@ -25,7 +64,7 @@ User-Facing Changes
 Individual components
 ~~~~~~~~~~~~~~~~~~~~~~
 
-- `renku-ui 3.12.21 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.12.2>`_
+- `renku-ui 3.12.2 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.12.2>`_
```

Comment:
uhm, is this correct? seems something weird going on here...

Reply:
seems like it should be `3.13.0` I think

Reply:
So this is correct.